From 290f6352f21627dc12a5a8e1a1141a230ed990a0 Mon Sep 17 00:00:00 2001
From: Chrissi^ <chris@tinyhost.de>
Date: Sun, 24 Feb 2019 15:22:53 +0100
Subject: [PATCH 07/38] gluon-mesh-batman-adv: Remove ebtables filter for bat0

This change alles meshed PARKER-Nodes to forward IPv6 to the next node.

Signed-off-by: Chrissi^ <chris@tinyhost.de>
---
 .../luasrc/lib/gluon/ebtables/300-radv-input-output             | 2 --
 1 file changed, 2 deletions(-)

diff --git a/package/gluon-mesh-batman-adv/luasrc/lib/gluon/ebtables/300-radv-input-output b/package/gluon-mesh-batman-adv/luasrc/lib/gluon/ebtables/300-radv-input-output
index 377d11cd..e69de29b 100644
--- a/package/gluon-mesh-batman-adv/luasrc/lib/gluon/ebtables/300-radv-input-output
+++ b/package/gluon-mesh-batman-adv/luasrc/lib/gluon/ebtables/300-radv-input-output
@@ -1,2 +0,0 @@
-rule 'INPUT -p IPv6 --ip6-protocol ipv6-icmp --ip6-icmp-type router-solicitation -i bat0 -j DROP'
-rule 'OUTPUT -p IPv6 --ip6-protocol ipv6-icmp --ip6-icmp-type router-advertisement -o bat0 -j DROP'
-- 
2.34.1

